frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-install-global.spec.ts

```diff
@@ -1,12 +1,47 @@
 import { checkErrors } from '../../../integration-tests-cypress/support';
+// import { checkErrors, testName } from '../../../integration-tests-cypress/support';
 import { modal } from '../../../integration-tests-cypress/views/modal';
 import { detailsPage } from '../../../integration-tests-cypress/views/details-page';
 
 const operatorName = 'Portworx Essentials';
+const operatorPkgName = 'portworx-essentials';
+
+describe(`Interacting with a global install mode Operator (${operatorName})`, () => {
+  const catalogSource = {
+    apiVersion: 'operators.coreos.com/v1alpha1',
+    kind: 'CatalogSource',
+    metadata: { name: 'console-e2e', labels: { 'olm-visibility': 'visible' } },
+    spec: {
+      sourceType: 'grpc',
+      image:
+        'quay.io/operator-framework/upstream-community-operators@sha256:10121664b6ab87b0bd36b0f0011bf0f9b0dd55b41080878058f3c1052e869ed5',
+      displayName: 'Console E2E Operators',
+      publisher: 'Red Hat, Inc',
+    },
+  };
+  const catalogNamespace = 'openshift-marketplace';
 
-xdescribe(`Interacting with a global install mode Operator (${operatorName})`, () => {
   before(() => {
     cy.login();
+    cy.exec(`echo '${JSON.stringify(catalogSource)}' | kubectl create -n ${catalogNamespace} -f -`);
+    // await new Promise((resolve) =>
```

Comment:
Not sure how to best check to see if the `packagemanifest` has been created or not using Cypress.  This is how we did it in Protractor.